### DISCUSSION TITLE
My friend passed away. We want to preserve his account, is there anything we can do?

### DISCUSSION THREAD

**Discussion by MrComet-Ha** [community/community#54412] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My friend passed away. I want to preserve his account forever under agreement with the bereaved family, is there anything the bereaved family or I can do in this case?

**Comment by radar-blip** [comment]
Here is some info on that matter: https://docs.github.com/en/site-policy/other-site-policies/github-deceased-user-policy
Hope it helps.

**Reply by MrComet-Ha** [reply]
thanks.
